### DISCUSSION TITLE
Are there SEO descriptions for individual pages?

### DISCUSSION THREAD

**Discussion by Destaq** [mmistakes/minimal-mistakes#2586] [Q&A] [answered]
I would like to add page descriptions for SEO on Google. I know that you can provide a site description with `description: "<description here>"` in the `_config.yml` file. However, this only provides a Google SEO description for the home page.

Is there any way to create these Google snippets/suggestions for individual pages or blog posts on the site?

By description I mean the small grey text under a Google search result (see below).
![image](https://user-images.githubusercontent.com/61620873/86121327-1ec1b380-bad6-11ea-9b8e-8090fae2d1cb.png)

**Comment by mmistakes** [comment, marked as answer]
This is mentioned in the documentation on the bottom of this page. https://mmistakes.github.io/minimal-mistakes/docs/posts/

![image](https://user-images.githubusercontent.com/1376749/86124949-e310f480-baa9-11ea-8cf4-d37f4b3087bc.png)

**TL/DR:** add text to `excerpt` front matter which will be used as a meta description in SERPs.
